### ISSUE TITLE
no generation info displayed after latest a1111 update

### ISSUE THREAD

**Issue by nickr61** [yfszzx/stable-diffusion-webui-images-browser#88]
After the latest update of Automatic1111 there is no longer any info displayed when an image is selected or any 'send to' buttons

**Comment by prudyray305** [comment]
Yeah, same here.

**Comment by atmogenic** [comment]
yep same here including the UI being a bit messed up on the txt2img page

**Comment by PapaBones** [comment]
Also some different ways the UI is behaving now:
You can no longer click on a page to see a full overview, it'll always stay on an image and you have to close them with the X.
Also no delete button anymore.

**Comment by nickr61** [comment]
OK there is a workaround until this is fixed, this puts your installation back to a previous branch, I just did it and everything now working as it was 
Open up a CMD prompt at the base of the stable-diffusion-webui folder, and type in "git checkout [a9eab23](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/a9eab236d7e8afa4d6205127904a385b2c43bb24)" and hit enter, that should do it. Then make sure to edit the webui-user.bat file if you added "git pull" so it doesn't update again. And just start as usual.

To revert to the master branch just do the same but type "git checkout master"

**Comment by nickr61** [comment]
To revert to the master branch just do the same but type  git checkout master

**Comment by aetherwu** [comment]
> OK there is a workaround until this is fixed, this puts your installation back to a previous branch, I just did it and everything now working as it was Open up a CMD prompt at the base of the stable-diffusion-webui folder, and type in "git checkout [a9eab23](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/a9eab236d7e8afa4d6205127904a385b2c43bb24)" and hit enter, that should do it. Then make sure to edit the webui-user.bat file if you added "git pull" so it doesn't update again. And just start as usual.
> 
> To revert to the master branch just do the same but type "git checkout master"

Looking for update and thanks a lot for your amazing contributions!

**Comment by zyd232** [comment]
It seems this repo has been abandoned. Now I'm using this fork: 
https://github.com/AlUlkesh/stable-diffusion-webui-images-browser
It works well with the updates for gradio 3.22.

**Comment by nickr61** [comment]
@zyd232 , interesting, OK I'll give that a try, thanks for pointing that out

**Comment by BlastedRemnants** [comment]
> It seems this repo has been abandoned. Now I'm using this fork: https://github.com/AlUlkesh/stable-diffusion-webui-images-browser

Thanks! Can confirm it seems to work so far, and feels familiar enough to be a painless change.

**Comment by xpgx1** [comment]
> It seems this repo has been abandoned. Now I'm using this fork: https://github.com/AlUlkesh/stable-diffusion-webui-images-browser It works well with the updates for gradio 3.22.

Works as intended, but there are still no generation parameters on display - at least here, for me. Could be related to something else, since my output library lies on a network share. 

Thank you for the fork mention, hadn't noticed it. In the meantime, I'll suggest using the "Diffusion Toolkit" tool from https://github.com/RupertAvery/DiffusionToolkit/ . It's not as integrated, obviously, but it provides you with generation info, reliably.
